pkg/lock/dir.go

```diff
@@ -22,6 +22,8 @@ import (
 
 var (
 	ErrLocked = errors.New("directory already locked")
+	ErrAbsent = errors.New("directory missing")
```

Comment:
+1, or if you want to make them explicitly specific to this package then IMHO  make the names more consistent with the stdlib e.g. `ErrNotExist = errors.New("directory does not exist")`, `ErrPermission`, ...